.github/workflows/cli-master.yml

```diff
@@ -6,11 +6,6 @@ on:
     paths-ignore:
       - 'docs/**'
       - '**/*.md'
-  pull_request:
-    branches: [master]
-    paths-ignore:
-      - 'docs/**'
-      - '**/*.md'
```

Comment:
Why delete this?

Reply:
This CI is used to test whether the Rockspec for **remotely pulling** the master branch can be installed.
It is meaningless for PR, assuming that a third-party library in Rockspec has been updated and a certain field has been adjusted, then PR cannot pass.